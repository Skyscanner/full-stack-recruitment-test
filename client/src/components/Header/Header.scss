@import '~bpk-mixins';

.Header {
  display: flex;
  padding: $bpk-spacing-sm;
  justify-content: space-between;
  align-items: center;
  background-color: $bpk-color-white;

  @include bpk-box-shadow-sm;

  &__hidden-text {
    @include bpk-visually-hidden;
  }

  &__logo-image {
    height: $bpk-spacing-base;
    vertical-align: top;
  }
}
