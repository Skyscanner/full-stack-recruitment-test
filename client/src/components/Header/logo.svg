<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 211 36" width="211" height="36" fill="#00b2d6"><path d="M70.187 14.115a.577.577 0 0 1-.765.365c-.969-.373-2.065-.64-3.09-.64-1.764 0-3.426.53-3.426 2.296 0 .954.53 1.449 2.227 1.98l1.236.387c2.473.776 3.533 1.907 3.533 4.096 0 3.778-2.65 6.358-7.452 6.358-1.887 0-3.501-.296-4.87-.773-.333-.116-.493-.486-.394-.825l.565-1.96c.094-.325.456-.479.77-.353 1.237.5 2.608.839 3.822.839 2.156 0 3.673-.742 3.673-2.474 0-1.024-.706-1.411-2.296-1.907l-1.343-.423c-2.367-.777-3.427-2.19-3.427-4.45 0-3.851 3.216-5.933 7.454-5.933 1.275 0 2.732.285 3.947.702.335.114.497.486.394.824l-.558 1.891zm8.347-10.082c.33 0 .575.3.514.623l-2.6 13.436c2.45-1.787 4.59-3.887 6.48-6.324a1.27 1.27 0 0 1 1.007-.494h2.828c.272 0 .423.307.267.53-1.859 2.621-3.986 5.04-6.237 6.993l3.582 9.017a.434.434 0 0 1-.402.592h-3.005a.915.915 0 0 1-.854-.595l-2.465-6.717c-.662.446-1.346.859-2.049 1.237l-1.046 5.451a.772.772 0 0 1-.753.624h-2.582a.523.523 0 0 1-.512-.624l4.49-23.126a.768.768 0 0 1 .756-.624h2.581v.001zm13.213 7.24c.33 0 .577.302.514.626l-1.75 9.019c-.142.67-.283 1.518-.283 2.332 0 1.058.6 1.766 2.013 1.766 1.378 0 3.001-.601 4.203-1.73l2.245-11.388a.774.774 0 0 1 .756-.624h2.582c.329 0 .575.3.511.625l-2.314 11.88c-.283 1.377-.6 2.931-.882 4.168C97.999 33.528 95.456 36 90.369 36c-1.12 0-2.235-.14-3.205-.4a.687.687 0 0 1-.494-.818l.442-1.804a.616.616 0 0 1 .738-.461c.733.175 1.571.304 2.343.304 3.005 0 4.768-1.272 5.404-4.732l.25-1.308c-1.307 1.13-3.252 1.872-5.087 1.872-2.968 0-4.557-1.483-4.557-3.85 0-1.413.283-2.968.6-4.59l1.61-8.313a.773.773 0 0 1 .755-.626h2.579v-.001zm23.306 2.855a.577.577 0 0 1-.765.366c-.968-.374-2.066-.642-3.09-.642-1.764 0-3.424.532-3.424 2.297 0 .954.527 1.449 2.223 1.98l1.237.387c2.472.776 3.533 1.907 3.533 4.097 0 3.778-2.65 6.358-7.452 6.358-1.885 0-3.502-.297-4.868-.774-.334-.116-.494-.486-.397-.825l.567-1.96c.093-.325.457-.479.77-.353 1.236.5 2.607.839 3.82.839 2.155 0 3.674-.742 3.674-2.474 0-1.024-.705-1.411-2.294-1.907l-1.346-.423c-2.365-.777-3.424-2.19-3.424-4.45 0-3.851 3.214-5.933 7.454-5.933 1.275 0 2.73.285 3.946.702.332.114.494.486.393.824l-.557 1.891zm15.075.387a.526.526 0 0 1-.735.323c-.93-.425-1.982-.668-3.181-.668-3.743 0-6.007 3.216-6.007 7.313.036 2.439.92 4.063 3.676 4.098 1.22.03 2.493-.217 3.644-.613.282-.096.59.101.556.396l-.243 2.058a1.044 1.044 0 0 1-.692.868c-1.28.445-2.675.681-4.254.681-4.593 0-6.747-2.613-6.747-6.924 0-6.108 3.462-11.335 10.101-11.335 1.61 0 2.97.3 4.075.775.343.151.52.533.415.893l-.608 2.135zm2.814-1.969a.92.92 0 0 1 .618-.85c1.692-.565 3.828-.984 5.685-.984 3.531 0 5.827 1.059 5.827 4.132 0 .635-.106 1.413-.281 2.26l-1.377 7.24c-.213 1.087-.108 1.695.362 1.933.288.145.53.422.464.739l-.207 1.017c-.08.392-.377.716-.769.8a6.464 6.464 0 0 1-1.264.14c-1.592 0-2.367-.743-2.472-2.05h-.106c-1.165 1.272-2.93 2.05-5.017 2.05-3.001 0-4.943-1.518-4.943-4.556 0-4.098 3.25-7.03 10.526-7.03h.955c.104-.53.174-1.06.174-1.448 0-1.413-.707-2.05-2.93-2.05-1.415 0-3.112.338-4.713.883a.432.432 0 0 1-.572-.421l.04-1.805zm6.762 7.595c-3.567 0-6.182 1.165-6.182 3.743 0 1.132.673 1.838 2.191 1.838 1.447 0 2.967-.742 3.92-1.731l.775-3.85h-.704zm12.01-8.867c.341 0 .608.293.574.634l-.115 1.204h.034c1.73-1.449 3.957-2.4 6.111-2.4 2.685 0 4.45 1.552 4.45 4.026 0 1.2-.175 2.188-.387 3.319l-1.859 9.726a.774.774 0 0 1-.754.624h-2.582a.521.521 0 0 1-.512-.624l1.714-8.879c.178-.882.32-1.872.32-2.472 0-1.306-.708-1.977-2.192-1.977-1.623 0-3.461.845-4.767 2.082l-2.142 11.246a.77.77 0 0 1-.752.624h-2.581a.523.523 0 0 1-.516-.624l3.045-15.884a.768.768 0 0 1 .752-.625h2.159zm17.485 0a.58.58 0 0 1 .575.634l-.117 1.204h.035c1.731-1.449 3.955-2.4 6.109-2.4 2.686 0 4.452 1.552 4.452 4.026 0 1.2-.178 2.188-.388 3.319l-1.86 9.726a.772.772 0 0 1-.754.624h-2.582a.525.525 0 0 1-.515-.624l1.72-8.879c.176-.882.317-1.872.317-2.472 0-1.306-.707-1.977-2.191-1.977-1.624 0-3.46.845-4.768 2.082l-2.14 11.246a.771.771 0 0 1-.755.624h-2.582a.525.525 0 0 1-.516-.624l3.047-15.884a.768.768 0 0 1 .755-.625h2.158zm16.53 10.208c-.06.517-.05 1.04.035 1.555.318 1.73 1.66 2.578 3.92 2.578 1.578 0 3.234-.243 4.689-.706.28-.089.57.126.532.418l-.235 1.955c-.046.382-.289.721-.654.849-1.6.559-3.41.84-5.11.84-4.873 0-7.24-2.084-7.24-6.816 0-4.099 1.34-7.877 4.239-9.89 1.448-1.026 3.25-1.553 5.51-1.553 4.097 0 5.898 2.328 5.898 5.792 0 1.193-.142 2.781-.471 4.388-.073.355-.402.59-.764.59h-10.349zm7.416-2.79c.18-.668.275-1.357.283-2.049 0-1.59-.705-2.72-2.506-2.72-2.614 0-4.205 1.908-4.769 4.769h6.992zm10.495-7.418a.59.59 0 0 1 .588.634l-.098 1.38h.07c1.308-1.59 3.217-2.402 5.017-2.402a7.2 7.2 0 0 1 .514.016c.337.021.527.376.426.697l-.715 2.285c-.113.365-.468.597-.85.613-1.784.077-3.508.587-4.674 1.406l-2.28 11.88a.773.773 0 0 1-.756.624h-2.58a.524.524 0 0 1-.515-.624l3.046-15.884a.77.77 0 0 1 .752-.625h2.055zM28.247 14.713c4.78-4.781 11.05-7.158 17.318-7.152l.05.001c.57 1.336.901 2.763.978 4.214a20.283 20.283 0 0 0-1-.025l-.028.002-.026-.002a20.127 20.127 0 0 0-14.322 5.933 20.126 20.126 0 0 0-5.934 14.321l.003.072H21.09c-.017-6.282 2.361-12.572 7.156-17.365l.001.001zm24.843 8.02c.03-.329.046-.662.046-.999 0-1.604-.369-3.12-1.023-4.473a16.159 16.159 0 0 0-6.548-1.371 16.06 16.06 0 0 0-11.43 4.715 16.063 16.063 0 0 0-4.716 11.473h4.21c-.018-3.08 1.132-6.162 3.475-8.505a11.87 11.87 0 0 1 8.25-3.473v-.003l.211.002.213-.002v.003c2.597.038 5.183.913 7.312 2.633zm-42.019-4.917l-.002-.017.002.017zm40.225 8.491a8.011 8.011 0 0 0-5.518-2.353v.002l-.213-.002-.212.002v-.002a8.014 8.014 0 0 0-5.517 2.353 8.02 8.02 0 0 0-2.35 5.772h4.04c.265-1.35.51-2.032 1.28-2.803a3.83 3.83 0 0 1 2.76-1.11 3.83 3.83 0 0 1 2.76 1.11c.263.264.473.558.642.869a10.095 10.095 0 0 0 2.851-3.258 7.998 7.998 0 0 0-.523-.58zm-34.17 5.7c0-14.844 11.382-27.026 25.895-28.301A12.408 12.408 0 0 0 34.164 0c-6.382 0-11.64 4.8-12.368 10.985a7.013 7.013 0 0 0-3.737-1.074c-3.89 0-7.432 3.892-6.988 7.905a7.562 7.562 0 1 0-4.603 14.183c.287.05.582.08.883.08h9.775v-.074.002zM11.06 17.718l.009.08-.009-.08z"></path></svg>
