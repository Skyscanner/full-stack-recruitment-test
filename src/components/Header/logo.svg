<svg height="259.35999" viewBox="0 0 1600 259.35999" width="1600" xmlns="http://www.w3.org/2000/svg"><g fill="#396fc0" transform="matrix(.13333333 0 0 -.13333333 0 259.36)"><path d="m1319 279.801c27.13-.012 54.26 7.019 78.54 21.07l288.31 166.449c58.3 33.629 125.82 47.739 192.69 40.102 350.31-40.371 596.55-106.582 669.09-127.684 15.04-4.359 31.12.66 41.02 12.934 11.87 14.769 27.03 38.379 37.85 72.277 10.81 33.36 12.26 60.922 11.21 79.922-.79 16.348-12 30.34-27.7 34.949-113.69 33.5-615.99 169.481-1291.01 169.481s-1177.324-135.981-1291.0156-169.481c-15.6953-4.609-26.90627-18.601-27.695337-34.949-1.054688-19 .394531-46.562 11.210937-79.922 10.8164-33.898 25.9805-57.508 37.8516-72.277 9.8906-12.274 25.9804-17.293 41.0195-12.934 72.5389 21.102 318.7769 87.313 669.0819 127.684 66.871 7.637 134.399-6.473 192.695-40.102l288.312-166.449c24.28-14.051 51.41-21.082 78.54-21.07"/><path d="m837.457 1215.44c15.844-27.44 41.309-45.86 69.672-53.46s59.629-4.38 87.07 11.46c27.441 15.85 45.861 41.31 53.461 69.67 7.6 28.37 4.38 59.63-11.46 87.07l-259.427 449.35c-8.312 14.4-24.507 20.26-41.148 19.04-20.504-1.64-49.727-12.26-83.883-31.98-34.148-19.71-57.957-39.72-69.633-56.65-9.379-13.8-12.39-30.76-4.082-45.15z"/><path d="m599.688 893.68c27.437-15.852 58.703-19.078 87.07-11.469 28.363 7.598 53.828 26.019 69.672 53.461 15.843 27.441 19.062 58.707 11.465 87.068-7.602 28.37-26.024 53.83-53.465 69.68l-449.348 259.42c-14.391 8.32-31.348 5.3-45.148-4.08-16.93-11.67-36.938-35.48-56.653-69.63-19.722-34.16-30.336-63.38-31.98-83.88-1.223-16.64 4.644-32.84 19.035-41.15z"/><path d="m1433.75 1374.99c0-31.68-12.85-60.36-33.61-81.13-20.77-20.77-49.46-33.61-81.14-33.61-31.69 0-60.37 12.84-81.14 33.61-20.76 20.77-33.61 49.45-33.61 81.13v518.87c0 16.61 11.09 29.79 26.12 37.06 18.58 8.82 49.2 14.24 88.63 14.24 39.44 0 70.05-5.42 88.63-14.24 15.03-7.27 26.12-20.45 26.12-37.06z"/><path d="m1800.54 1215.44c-15.84-27.44-41.31-45.86-69.67-53.46-28.37-7.6-59.63-4.38-87.07 11.46-27.44 15.85-45.87 41.31-53.46 69.67-7.61 28.37-4.38 59.63 11.46 87.07l259.43 449.35c8.31 14.4 24.5 20.26 41.15 19.04 20.5-1.64 49.72-12.26 83.87-31.98 34.16-19.71 57.96-39.72 69.64-56.65 9.38-13.8 12.39-30.76 4.09-45.15z"/><path d="m2038.32 893.68c-27.45-15.852-58.71-19.078-87.07-11.469-28.37 7.598-53.84 26.019-69.68 53.461-15.84 27.441-19.07 58.707-11.46 87.068 7.6 28.37 26.02 53.83 53.46 69.68l449.35 259.42c14.39 8.32 31.35 5.3 45.15-4.08 16.93-11.67 36.93-35.48 56.65-69.63 19.72-34.16 30.33-63.38 31.98-83.88 1.22-16.64-4.65-32.84-19.04-41.15z"/><path d="m4906.46 1250.29h215.94c7.93 0 15.04-4.86 17.92-12.24l215.31-551.679 208.83 551.489c2.83 7.48 9.99 12.43 17.99 12.43h212.1c13.86 0 23.17-14.22 17.63-26.93l-528.19-1211.1881c-3.04-6.99221-9.94-11.511744-17.56-11.542994l-186.88-.628906c-13.82-.0507813-23.18 14.0703-17.76 26.7813l180.24 422.8597-353.07 773.429c-5.82 12.73 3.49 27.22 17.5 27.22"/><path d="m7628.5 822.52c0-147.782 91.51-261.54 237.08-261.54 145.56 0 240.46 98.309 240.46 250.579 0 152.269-91.59 268.301-246.37 268.301-146.35 0-231.17-114.262-231.17-257.34zm471.63 305.43v103.1c0 10.63 8.61 19.24 19.24 19.24h191.01c10.63 0 19.24-8.61 19.24-19.24v-821.269c0-10.633-8.61-19.242-19.24-19.242h-191.01c-10.63 0-19.24 8.609-19.24 19.242v108.16c-40.9-63.402-130.81-145.953-276.74-145.953-279.86 0-425.23 213.34-425.23 452.223 0 310.989 215.99 444.639 418.48 444.639 132.99 0 225.5-62.68 283.49-140.9"/><path d="m8502.96 409.781v821.269c0 10.63 8.61 19.24 19.24 19.24h191.01c10.63 0 19.24-8.61 19.24-19.24v-111.53c45.5 78.33 128.99 149.33 272.52 149.33 148 0 312.18-72.71 312.18-406.67v-452.399c0-10.633-8.61-19.242-19.24-19.242h-191.02c-10.62 0-19.23 8.609-19.23 19.242v449.02c0 80.609-19.82 218.529-162.84 218.529s-192.37-125.295-192.37-241.299v-426.25c0-10.633-8.61-19.242-19.24-19.242h-191.01c-10.63 0-19.24 8.609-19.24 19.242"/><path d="m9483.71 409.781v821.269c0 10.63 8.61 19.24 19.23 19.24h191.02c10.62 0 19.23-8.61 19.23-19.24l.01-111.53c45.49 78.33 128.98 149.33 272.52 149.33 147.98 0 312.18-72.71 312.18-406.67v-452.399c0-10.633-8.6-19.242-19.2-19.242h-191.1c-10.6 0-19.2 8.609-19.2 19.242v449.02c0 80.609-19.8 218.529-162.83 218.529-143.02 0-192.37-125.295-192.37-241.299l-.01-426.25c0-10.633-8.61-19.242-19.23-19.242h-191.02c-10.62 0-19.23 8.609-19.23 19.242"/><path d="m11491.9 1250.29h186c10.6 0 19.2-8.61 19.2-19.24v-169.75c37.5 168.8 181.9 226.23 288 202.24 8.8-1.97 14.9-9.84 14.9-18.79v-187.41c0-12.16-11.2-21.16-23.1-18.82-195.5 38.3-274.7-66.922-274.7-215.149 0-105.082 0-344.281 0-413.641 0-10.621-8.6-19.191-19.3-19.191h-191c-10.6 0-19.2 8.609-19.2 19.242v821.269c0 10.63 8.6 19.24 19.2 19.24"/><path d="m3639.08 1063.89c-58.86 20.38-90.37 31.29-138.59 47.99s-131.44 64.57-131.44 141.72 53.3 124.94 159.47 124.94c97.98 0 152.07-47.93 187.3-123.87 4.79-10.33 17.28-14.46 27.14-8.77l156.88 90.58c8.56 4.94 12.05 15.56 7.97 24.56-47.34 104.34-157.79 229.26-377.18 229.26-244.51 0-396.94-145.41-396.94-335.79s125.67-289.1 295.44-346.85c61.74-21.012 87.53-29.781 139.75-47.539 99.93-33.992 146.09-78.992 146.09-141.551 0-62.57-39.04-135.66-195.32-135.66-142.38 0-193.06 75.262-221.36 147.059-4.28 10.871-17.31 15.293-27.43 9.453l-164.25-94.832c-8.34-4.809-11.81-15.059-8.23-23.988 62.69-156.243 235.77-248.614 430.56-248.614 228.08 0 420.59 116.551 420.59 350.891 0 234.359-251.6 320.641-310.45 341.011"/><path d="m4899.12 614.82c-5.69 9.84-18.67 12.668-27.86 5.969-73.99-53.957-161.51-55.027-226.56 46.672-55.12 86.187-122.05 190.84-122.05 190.84l289.56 360.709c10.11 12.59 1.14 31.28-15 31.28h-226.59c-6.15 0-11.93-2.94-15.55-7.91l-250.68-344.189v693.129c0 10.62-8.61 19.23-19.23 19.23h-191.02c-10.62 0-19.24-8.61-19.24-19.23v-1181.539c0-10.633 8.62-19.23 19.24-19.23h191.02c10.62 0 19.23 8.597 19.23 19.23v384.059s128.39-202.219 176.09-277.371c65.42-103.059 158.87-144.481 258.14-144.481 94.65 0 155.51 25.453 228.39 95.891 6.37 6.16 7.78 15.871 3.35 23.543z"/><path d="m6271.16 887.48c-61.1 22.579-82.05 30.329-128.58 47.52-46.54 17.199-86.6 42.02-86.6 79.17 0 37.14 33.93 78.35 103.78 78.35 60.98 0 98.2-24.33 118.68-66.01 4.91-10 17.2-13.82 26.85-8.25l134.25 77.51c9.21 5.31 12.35 17.06 7.06 26.28-36.75 63.99-109.91 146.8-285.15 146.8-215.22 0-325.68-124.35-325.68-261.55 0-137.198 112.39-201.859 215.15-243.339 116.14-46.91 123.96-50.063 138.19-55.813 35.86-14.468 78.64-36.82 78.64-78.218 0-41.418-46.87-77.391-120.65-77.391-67.91 0-143.77 27.691-175.14 105.289-4.38 10.852-17.29 15.391-27.43 9.531l-135.42-78.187c-8.22-4.742-11.82-14.781-8.38-23.621 40.04-102.711 157.38-193.563 346.37-193.563 205.7 0 342.55 109.864 342.55 264.082 0 154.219-102.59 208.571-218.49 251.41"/><path d="m7044.72 1062.99c85.6 0 149.41-25.97 208.09-68.748 9.28-6.769 22.36-4.133 28.1 5.818l78.87 136.6c5.06 8.79 2.44 19.95-5.96 25.63-81.93 55.33-182.97 106.56-320.07 106.56-135.21 0-261.45-37.44-351.42-127.4-89.96-89.96-131.19-196.391-131.19-323.149 0-153.332 62.9-250.192 132.04-319.34 69.14-69.141 192.63-126.133 350.57-126.133 142.64 0 248.57 58.774 319.93 107.192 8.34 5.671 10.83 16.789 5.79 25.519l-78.34 135.691c-5.68 9.848-18.71 12.77-27.86 6.008-50.81-37.566-125.01-68.539-208.55-68.539-93.25 0-263.25 53.352-263.25 242.141 0 188.78 169.22 242.15 263.25 242.15"/><path d="m10887.8 1084.07c109.7 0 210.1-65.9 225.3-184.769h-459.8c29.8 128.899 124.9 184.769 234.5 184.769zm37.2-521.41c-77 0-149.3 17.86-202.1 60.328-52.8 42.461-77.2 85.633-77.2 137.102h683.4c10.8 0 19.4 8.941 19.2 19.742-4 203.039-69.7 298.648-133.5 362.458-65.9 65.84-178.3 126.56-325.3 126.56-146.9 0-261.6-54.08-338.3-130.78-76.7-76.69-134.1-173.625-134.1-321.449 0-147.832 59.5-244.769 132.4-317.66 72.9-72.883 191-126.949 361-126.949 161.6 0 296.2 57.066 394.5 168.636 8 9.102 5.6 23.321-4.9 29.383l-136.1 78.598c-7.2 4.141-16.3 3.301-22.4-2.277-68.8-62.821-154.9-83.692-216.6-83.692"/></g></svg>