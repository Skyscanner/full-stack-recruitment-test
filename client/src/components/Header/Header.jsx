import React from 'react';

import logo from './logo.svg';
import STYLES from './Header.scss';

const c = className => STYLES[className] || 'UNKNOWN';

const Header = () => (
  <header className={c('Header')}>
    <a href="/">
      <span className={c('Header__hidden-text')}>Skyscanner</span>
      <img className={c('Header__logo-image')} alt="Skyscanner" src={logo} />
    </a>
    {/* TODO burger menu icon */}
  </header>
);

export default Header;
