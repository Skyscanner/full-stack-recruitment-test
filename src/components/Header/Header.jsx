import React from 'react';

import logo from './logo.svg';
import STYLES from './Header.scss';

const getClassName = className => STYLES[className] || 'UNKNOWN';

const Header = () => (
  <header className={getClassName('Header')}>
    <a href="/">
      <span className={getClassName('Header__hidden-text')}>Skyscanner</span>
      <img
        className={getClassName('Header__logo-image')}
        alt="Skyscanner"
        src={logo}
      />
    </a>
  </header>
);

export default Header;
