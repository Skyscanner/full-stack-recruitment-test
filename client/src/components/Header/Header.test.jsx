import React from 'react';
import TestRenderer from 'react-test-renderer';

import Header from './Header';

describe('Header', () => {
  it('should render correctly', () => {
    const tree = TestRenderer.create(<Header />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});

