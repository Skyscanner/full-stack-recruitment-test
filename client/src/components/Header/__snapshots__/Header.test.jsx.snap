// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly 1`] = `
<header
  className="Header"
>
  <a
    href="/"
  >
    <span
      className="Header__hidden-text"
    >
      Skyscanner
    </span>
    <img
      alt="Skyscanner"
      className="Header__logo-image"
      src="logo.svg"
    />
  </a>
</header>
`;
